src/dataExtraction/wsLoad/csFlatFiles_to_ws.py

```diff
@@ -20,9 +20,6 @@
 import Bio.SeqIO
 import Bio.SeqFeature
 import urllib2
-import logging
-import warnings
-import re
 
```

Comment:
Do you or anyone else use this script?  It may be deprecated.  If it is I won't worry about this diff.